### PR TITLE
feat: expand MetaInfo to udf1-15 and add tests

### PR DESCRIPTION
**Pull request by 282000ypk**
- Expand MetaInfo from udf1-5 to udf1-15 with constructor params and getters
- Add MetaInfoTest with 4 tests covering all fields, null defaults, and getters
- Bump SDK_VERSION 2.0.0 -> 2.0.1
- Added PrefillUserLoginDetails support in standardcheckout

### CODE REVIEW

File: src/phonepe/sdk/pg/payments/v2/models/request/MetaInfo.php [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+		$this->udf11 = $udf11;
+		$this->udf12 = $udf12;
+		$this->udf13 = $udf13;
+		$this->udf14 = $udf14;
+		$this->udf15 = $udf15;
+	}
+
+	private function validateUdf1To10(string $field, ?string $value): void
+	{
+		if ($value !== null && strlen($value) > self::UDF1_TO_10_MAX_LENGTH) {
```
Comment (sk4x0r) [inline src/phonepe/sdk/pg/payments/v2/models/request/MetaInfo.php]: **`strlen()` vs `mb_strlen()`** — `strlen()` counts **bytes**, not characters. Since udf1–10 allow arbitrary content (including emoji/multibyte chars), a 256-**character** UTF-8 string could exceed 256 bytes and get wrongly rejected.

For example, `strlen("🎉")` returns `4`, not `1`.

Consider using `mb_strlen($value, "UTF-8")` if the limit is character-based, or document explicitly that the limit is byte-based.

Comment (282000ypk) [reply src/phonepe/sdk/pg/payments/v2/models/request/MetaInfo.php]: Added the recommended fix
